src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -144,7 +144,7 @@
             @NotNull @PathVariable(value = "nomsNumber") final String nomsNumber) {
```

Comment:
Could you do the same here for requests by nomsNumbers to make it consistent. I think it is an important feature that we should have for prison based services as well